Chargebee/Classes/Result/CBError.swift

```diff
@@ -51,7 +45,7 @@ public struct CBErrorDetail: Decodable, ErrorDetail {
     public let type: String?
     public let apiErrorCode: String?
     public let param: String?
-    public let httpStatusCode: Int
+    public let httpStatusCode: Int?
```

Comment:
Why do we need to make this optional? Also this is a public contract and we wouldn't want to change this unless absolutely needed.

Reply:
this i reverted back  its not required to make it optional thanks